### PR TITLE
swift-format 0.50300.0

### PR DESCRIPTION
**Pull request by cprecioso**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/swift-format.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class SwiftFormat < Formula
     sha256 "d8f72c33efc125e2904e1bec2c8942cca75d75cf81dcab7fcf08ba124af16170" => :catalina
   end
 
-  depends_on xcode: ["11.4", :build]
+  depends_on xcode: ["12"]
```
Comment (cprecioso) [inline Formula/swift-format.rb:16]: I removed the "build" requirement,  because the final binary does call out to the locally installed Swift toolkit

**Comment by SMillerDev** [comment]
```
 ==> swift build --disable-sandbox -c release
Fetching https://github.com/apple/swift-argument-parser.git
Fetching https://github.com/apple/swift-syntax
error: the package dependency graph could not be resolved; unable to find any available tag for the following requirements:
    https://github.com/apple/swift-argument-parser.git @ 0.3.1..<0.4.0
```

**Comment by cprecioso** [comment]
>  ==> swift build --disable-sandbox -c release
> Fetching https://github.com/apple/swift-argument-parser.git
> Fetching https://github.com/apple/swift-syntax
> error: the package dependency graph could not be resolved; unable to find any available tag for the following requirements:
>     https://github.com/apple/swift-argument-parser.git @ 0.3.1..<0.4.0

It does work on my computer, and there is indeed a 0.3.1 release, maybe there is something wrong with where you tried it?

**Comment by cprecioso** [comment]
Also, I don't know why the CI failed. Is the OS it's testing on already on Xcode 12?

**Comment by SMillerDev** [comment]
> Also, I don't know why the CI failed. Is the OS it's testing on already on Xcode 12?

The error I posted is the error in the CI logs

**Comment by caipre** [comment]
Is it possible to re-run these tests?

**Comment by cprecioso** [comment]
Done

**Comment by cprecioso** [comment]
@SMillerDev the build passes on 10.15 and not on the previous versions (this is to be expected since Xcode 12 needs macOS 10.15). Is it possible to merge now?

**Comment by cprecioso** [comment]
@chenrui333 Sorry to ping you, I saw you merged #63928. Maybe you'd be able to tell me what are the blockers to merge this?

**Comment by chenrui333** [comment]
@cprecioso maybe the build is not green? let me take a peek.

**Comment by cprecioso** [comment]
For those who are watching the PR, while this is being merged, I uploaded the updated `swift-format` formula [to my tap](https://github.com/cprecioso/homebrew-tap) (no guarantees of keeping it up-to-date but it shouldn't take much until it is merged here).

```sh
$ brew install cprecioso/tap/swift-format
```

**Comment by lstomberg** [comment]
I posted about the issue we are experiencing trying to update this bottle [here](https://github.com/Homebrew/homebrew-core/pull/65134#issuecomment-730464674).

**Comment by bayandin** [comment]
I guess this one can be closed since https://github.com/Homebrew/homebrew-core/pull/65134 has been merged

**Pull request closed without merging**